types/angular-cookies/index.d.ts

```diff
@@ -41,19 +52,27 @@ declare module 'angular' {
             * The cookie will be available only in secured connection.
             */
             secure?: boolean;
+            /**
+             * Prevents the browser from sending the cookie along with cross-site requests.
+             * Accepts the values "lax" and "strict".
+             */
+            samesite?: string;
         }
 
         /**
-         * CookieService
-         * see http://docs.angularjs.org/api/ngCookies.$cookies
+         * CookiesService
+         * see https://code.angularjs.org/1.3.20/docs/api/ngCookies/service/$cookies
          */
         interface ICookiesService {
+            /**
+             * DEPRECATED
+             */
             [index: string]: any;
```

Comment:
and this lines should be probably removed as well:
https://github.com/angular/angular.js/blob/e55d352e942465479fa9f93b566db20a96b4cf15/docs/content/guide/migration.ngdoc#L453
only public setters/getters are now possible I thik according to that line:
>Note that
for object values you need to use the `putObject` & `getObject` methods, as
`get`/`put` will not correctly save/retrieve the object values.